<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImWidget</class>
 <widget class="QWidget" name="ImWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,3,0,1,0">
   <property name="margin">
    <number>9</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="te_input"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_tip">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pb_sendMsg">
     <property name="text">
      <string>sendMsg</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
